Header header
geometry_msgs/Vector3 robot_pose
geometry_msgs/Vector3 pedestrian_pose
int16 pedestrian_id
std_msgs/Bool collision_label
std_msgs/Bool success_label
int16 success_type

